src/components/connection-prompt/MicrobitWearingInstructionDialog.svelte

```diff
@@ -38,7 +38,7 @@
           <p>{$t('connectMB.wearingSetup.requirements1')}</p>
         </div>
         <div class="flex flex-col items-center gap-y-2">
-          <img src={microbitStrapImage} class="w-150" alt="" />
+          <img src={microbitStrapImage} class="w-175" alt="" />
```

Comment:
Why make the strap image wider?

Reply:
The perils of find and replace. Reverted here -> [2c8fdbb](https://github.com/microbit-foundation/ml-trainer/pull/114/commits/2c8fdbbee25653f3fa42f3e5ea4069dd48f18b34).